svc-managers/managers/managers.go

```diff
@@ -154,7 +154,7 @@ func (e *ExternalInterface) GetManagers(req *managersproto.ManagerRequest) respo
 					return resp
 				}
 			}
-			err = e.DB.UpdateManagersData(req.URL, managerData)
+			err = e.DB.UpdateManagersData(req.URL, managerData, "Managers")
```

Comment:
We are updating managers data, do we need to pass table name, isn't implicit?

Reply:
Same function is used for other table updates under manager api

Reply:
Addressed comments